main.go

```diff
@@ -19,10 +19,10 @@ import (
 	"sort"
 	"strings"
 
-	"github.com/maxmind/mm-geofeed-verifier/verify"
+	"github.com/maxmind/mm-geofeed-verifier/v2/verify"
 )
 
-const version = "2.2.1"
+const version = "2.3.1"
```

Comment:
I suspect I still don't completely understand the correct update path here, but if we are increasing the minor version the patch version should go back to 0, making this `2.3.0`.